operator/tests/securedcluster/basic-sc/40-assert.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: central-prometheus-k8s
```

Comment:
This should be `secured-cluster-prometheus-k8s`, right? I think that's what @ludydoo meant.

Reply:
Added, do you think I should remove `central` resources and keep them only in central tests?

Reply:
Yes, they are unrelated. If one only installs the secured cluster CR, then no `central` resources are added. I'm surprised the assert worked tbh, maybe worth looking into?

Reply:
Right, that was my mistake. In one place there was `Central` where it should be `SecuredCluster`